src/data/tools.json

```
@@ -1311,5 +1327,45 @@
     "url": "https://www.pngegg.com/",
     "description": "A great place to find a wide variety of images you would desire for your designs.",
     "tag": [ "design assets", "design", "image"]
+  },
+  {
+    "id": 147,
+    "title": "3D icons",
+    "url": "https://3dicons.co/",
+    "description": "Beautifully crafted open source 3D icons.",
+    "logo": "3dicons.png",
+    "tag": [ "icons", "iconography", "design", "design assets" ]
+  },
+  {
+    "id": 148,
+    "title": "snippet generator",
+    "url": "https://snippet-generator.app/",
+    "description": "Snippet generator for Visual Studio Code, Sublime Text and Atom.",
+    "tag": [ "snippet", "tools", "utility", "generator" ]
+  },
+  {
+    "id": 149,
+    "title": "clientdb",
+    "url": "https://clientdb.dev/",
+    "description": "Build lightning fast apps with instantaneous interactions.",
+    "logo": "clientdb.png",
+    "tag": [ "database", "tool"]
+  },
+  {
+    "id": 150,
+    "title": "snoopForms",
+    "url": "https://snoopforms.com/",
+    "description": "The Open-source Typeform Alternative.",
+    "logo": "snoopForms.png",
```

nit: for consistency:
```suggestion
    "logo": "snoop-forms.png",
```

In other words, let’s always use lowercase for filenames.